requirements.txt

```diff
@@ -8,3 +8,4 @@ PyYAML
 paramiko
 cryptography
 setuptools
+vcrpy
```

Comment:
These are requirements for ansible, not for tests. No new requirements should be added here.